### PR TITLE
A simplified take on the about section 'table-like' divs and comment cleaning

### PR DESCRIPTION
**Pull request by koksyncho**
The layout of the divs is now dependant on the col-1 to col-12 grid system we made earlier so it is much simpler to change around.
Also most of the comments in the main style css file have been removed (the comments like /* About Section */ have been left so that we can orient easier).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h2 class="top-subheader">test case management system</h2>
                 </div>
             </div>
         </header>
-        <!-- About Section -->
+         <!-- About Section -->
```
Comment (atodorov) [inline index.html:60]: Remove leading space.

File: index.html [hunk 2]
```diff
@@ -67,59 +67,68 @@ <h2 class="top-subheader">test case management system</h2>
...
-
-            <div class="rTable secondary">
-                <div class="rTableBody">
-                    <div class="rTableRow">
-                        <div class="rTableHead">Everyday testing</div>
-                        <div class="rTableHead">Test management</div>
-                        <div class="rTableHead">Reporting</div>
-                        <div class="rTableHead">Integration</div>
+            <div class="secondary">
+                <div class="col-12">
```
Comment (atodorov) [inline index.html:71]: What happens if you remove the wrapping col-12 divs? Shouldn't 4 8 col-3 divs have the same visual layout even without the wrapping col-12 div ?

**Review by atodorov — CHANGES_REQUESTED** [review]
LTGM, minor changes requested

**Comment by koksyncho** [comment]
Thanks, I will look into removing the unnecessary code!

**Pull request closed without merging**